### ISSUE TITLE
null stdout inside vagrant

### ISSUE THREAD

**Issue by wyhysj** [ansible/ansible#10440]
##### Issue Type:

Bug Report
##### Ansible Version:

1.8.4
##### Environment:

Ubuntu 14.04 (vm on osx)
##### Summary:

After several play I realize it's "echo -e " cause the problem, but anyway running on OS X is fine, switch to my vm running on ubuntu, the stdout is NULL
##### Steps to reproduce:

```
    - name: test
      command: echo -e '\033[0;31m'"ColorRenderedOutput"'\033[0m'
```
##### Expect results:

ColorRenderedOutput    -- works fine on OS X
##### Actual results:

But actually running scripts (shell/command/raw/script) give me NULL stdout

**Comment by bcoca** [comment]
could you fill in the template https://github.com/ansible/ansible/blob/devel/ISSUE_TEMPLATE.md? we need more information to figure out what is going on.

**Comment by wyhysj** [comment]
Issue Type: Bug Report
Ansible Version: 1.8.4
Environment: Ubuntu 14.04 (vm on osx)
Summary:
After several play I realize it's "echo -e " cause the problem, but anyway running on OS X is fine, switch to my vm running on ubuntu, the stdout is NULL

Steps to reproduce:

    - name: test
      command: echo -e '\033[0;31m'"ColorRenderedOutput"'\033[0m'

Expect results:

ColorRenderedOutput    -- works fine on OS X

Actual results:

""

**Comment by jctanner** [comment]
The null output was corrected in 1.9 and above ...

```
changed: [el6host] => {"changed": true, "cmd": ["echo", "-e", "\\033[0;31mColorRenderedOutput\\033[0m"], "delta": "0:00:00.002379", "end": "2016-10-31 21:52:20.882352", "rc": 0, "start": "2016-10-31 21:52:20.879973", "stderr": "", "stdout": "[0;31mColorRenderedOutput[0m", "warnings": []}
```
